peps/pep-0751.rst

```diff
@@ -997,6 +1074,13 @@ lock files at least support single-use lock files. Neither type of lock file
 is better or worse than the other, it just changes how much can be written down
 in a single file (which can influence how manageable).
 
+Lock files that follow this PEP can be installed by any installer that
+implements the specification. This allows users of a lock file to not not be
```

Comment:
Nit: "not not"

Reply:
Thanks for noticing! I will fix it in a PR later today.